packages/@rnw-scripts/doxysaurus/src/config.ts

```diff
@@ -0,0 +1,135 @@
+/**
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ *
+ * @format
+ */
+
+//
+// Configuration for the Doxysaurus tool.
+//
+
+import path from 'path';
+import {promises as fs, existsSync} from 'fs';
+
+// Definition of a config.
```

Comment:
Nit: JS/TS uses JSDoc/TSDoc, which use JavaDoc style comments. Comments should be in the style of:

```js
/**
 * definition of a config
 */
```

VSCode will fill these in for you if you just add `/**` and hit enter.

Reply:
Same for other places. This style is used pretty much everywhere. Classes, props, etc.

Reply:
Do we generate docs or it is mostly for IntelliSense?

Reply:
We don't generate docs right now. TSDoc/JSDoc/JavaDoc comment here would imo be mostly for convention, since it's followed throughout the ecosystem. IntelliSense comments do actually work fine with "//" comments.

Reply:
Changed to use JSDoc comments in commit https://github.com/microsoft/react-native-windows/pull/7044/commits/9461a88b5325461887832ad381c21ad676ceea6d